_amenities/pools/pool-rules.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Pool Rules
+date: 2019-08-05 08:18
+author: Klahanie
+comments: true
+categories: [pool]
+show-title: true
+---
+The rules and requests made by lifesaving personnel must always be followed.
+
+The lifesaving personnel are in complete charge of the pool area and have the authority to ask anyone to leave the pool if that person disobeys the safety or general pool rules, or if his/her conduct is detrimental to the common pleasure of other patrons enjoying the pool.
+
+Admission to the pool may be denied for any of the following reasons:
+
+* If you appear to be under the influence of alcohol and/or drugs
+* If you have open sores, wounds, skin infections, and/or nasal or ear discharge
+* Any condition or evidence, which in the opinion of the manager will jeopardize the health and safety of the membership
+
+Children under 5 years old or non-swimmers must be supervised by a responsible person, who is in the water and remains within arms reach of the child at all times.
+
+No person unable to swim across the pool may use the pool unattended.  Any adult or child may have to demonstrate his/her swimming ability before entering deep water.
+
+Any child in a life jacket must be accompanied by an adult in the water, within arms reach.  No one in a life jacket will be allowed in the deep end or to use the slide. LIFE JACKETS MUST BE U.S. COAST GUARD APPROVED.  There is a stamp on the inside of the life jacket to show that it is CG Approved.
+
+* Infants/children not toilet trained must wear swim diapers under swim suits in order to prevent contamination.
+* Children must have their diapers changed on changing tables in the locker rooms.
+* Everyone must take a cleansing shower before using the pool.
+* All accidents should be reported to the lifesaving personnel immediately.  A first aid kit is available in the guard office.
+* In case of emergency, a 9-1-1 phone is located next to the boy's locker room door facing the pool deck.
+* NO RUNNING on the pool deck or in restrooms.
+* NO HORSEPLAY or ROUGH-HOUSING.
+* Do not hang on the lane lines or the rope.
+* Do not open skimmers.
+* Starting blocks may be used only in scheduled practices, competitions, and instruction when supervised by a coach or instructor.
+* Kickboards are for lap swimming, swim team, and swim lessons ONLY.
+* Jumps into the pool must be feet first and facing the pool.  NO SPINS, FLIPS, OR DIVES.
+* Inflatable beach balls or sponge balls are allowed during pool non-crowdedconditions and at the manager's discretion.

```

Comment:
This page could use a rewrite.
"non-crowded conditions"

Reply:
Agreed, but we may be best off doing that separate from the initial replacement of the website. Happy to let the committee take the lead here.